packages/round-manager/src/context/round/FinalizeRoundContext.tsx

```diff
@@ -225,10 +230,8 @@ async function finalizeToContract(
       payload: { finalizeRoundToContractStatus: ProgressStatus.IN_PROGRESS },
     });
 
-    const merkleRoootInBytes = ethers.utils.formatBytes32String(merkleRoot);
-
     const encodedDistribution = encodeDistributionParameters(
-      merkleRoootInBytes,
+      merkleRoot,
       distributionMetaPtr
     );
```

Comment:
@thelostone-mc  finalizeRoundToContract function which is called after encoding parameters, calls updateDistribution function via roundImplementation. 
updateDistribution has been removed in roundImp v2.  So this needs to be updated as well for this to work